app/javascript/dashboard/components-next/dialog/Dialog.vue

```diff
@@ -101,25 +108,27 @@ defineExpose({ open, close });
           <slot name="form">
```

Comment:
This slot should not have a name. It can be the default one. Otherwise we will have to add `<template #form>` everywhere we're using the dialog